workflow/scripts/compare_pathways.py

```
@@ -1,6 +1,9 @@
 import polars as pl
 import polars.selectors as cs
 import altair as alt
+import sys
+
+sys.stderr = open(snakemake.log[0], "w")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)